Config.uk

```diff
@@ -80,6 +80,12 @@ config LIBPYTHON3_EXTENSION_ZLIB
 	bool "zlib"
 	default y
 endif
+
+if LIBPYTHON_NUMPY
+config LIBPYTHON3_EXTENSION_NUMPY
```

Comment:
Why not do a `select LIBPYTHON_NUMPY` if `LIBPYTHON3_EXTENSION_NUMPY` is selected?

Reply:
This follows the convention used for all other extensions, and I don't see numpy as special enough to break the pattern.
In addition, having the numpy extension option show up irrespective of whether numpy is present in the build or not is definitely misleading, and if the default remains 'y' (as is the case for all other extensions) the build will fail at link time.